docs/import_specifications.ADR.md

```diff
@@ -135,43 +141,69 @@ checking, including for missing or unknown input parameters. Most error checking
 in the bulk import cell configuration tab like other uploads, allowing for a consistent user
 experience.
 
-### Missing files
+### Error handling
+
+The StS will return errors on a (mostly) per input file basis, with a string error code for
+each error type. Currently the error types are:
+
+* `cannot_find_file` if an input file cannot be found
+* `cannot_parse_file` if an input file cannot be parsed
+* `unexpected_error` if some other error occurs
+
+The HTTP code returned will be:
+
+* 500 if all errors are `unexpected_error`
+* 404 if at least one error is `cannot_find_file`
+* 400 if at least one error is `cannot_parse_file`
```

Comment:
This is going from "you're the least screwed" to "you're totally screwed". 500 implies you can try later when the server is fixed. 404 implies you can try later if you add the missing file. 400 means you're boned and you have to delete and start over.